### PR TITLE
Swift 5 support

### PR DESCRIPTION
**Pull request by GoranLilja**
Hey! I migrated the TinyKeychain project to Swift 5. Not too many changes, really. Mainly in the `debugDescription`s for the `Error`s. Let me know if you want me to change anything.

### CODE REVIEW

**Comment by swain** [comment]
Thanks so much for your contribution! Great to see other people getting involved--reviewing now!

File: TinyKeychain/KeychainKey.swift [hunk 1]
```diff
@@ -36,7 +36,7 @@ public extension Keychain {
      ```
      
      */
-    public struct Key<Object: Codable> {
+    struct Key<Object: Codable> {
```
Comment (swain) [inline TinyKeychain/KeychainKey.swift:39]: Is this change necessary as part of the Swift 5 update?

Comment (GoranLilja) [reply TinyKeychain/KeychainKey.swift:39]: Not necessary, but it's redundant since the extension itself is marked as `public`. Xcode gives you a warning when trying to compile using Swift 5.

`'public' modifier is redundant for generic struct declared in a public extension`

Comment (swain) [reply TinyKeychain/KeychainKey.swift:39]: Great catch!
